My_GOT_app/app/src/main/java/com/example/my_got_app/LannisterActivity.java

```diff
@@ -39,26 +48,35 @@ protected void onCreate(Bundle savedInstanceState) {
 
     }
 
+    /**
+     * Initializes the Grid Layout using RecyclerView
+     * Initializes an ArrayList called lannHouse using prepareList
+     * Sets the list adapter
+     */
     private void initViews(){
         RecyclerView recyclerView = (RecyclerView)findViewById(R.id.list);
         recyclerView.setHasFixedSize(true);
         RecyclerView.LayoutManager layoutManager = new GridLayoutManager(getApplicationContext(),2);
         recyclerView.setLayoutManager(layoutManager);
 
-        ArrayList<Person> starkHouse = prepareData();
-        ListAdapter adapter = new ListAdapter(starkHouse, getApplicationContext());
+        ArrayList<Person> lannHouse = prepareList();
+        ListAdapter adapter = new ListAdapter(lannHouse, getApplicationContext());
         recyclerView.setAdapter(adapter);
     }
 
-    private ArrayList<Person> prepareData(){
-        ArrayList<Person> starks = new ArrayList<>();
+    /**
+     * Sets the data in a list
+     * @return an ArrayList containing a Person object for each family member
+     */
+    private ArrayList<Person> prepareList(){
+        ArrayList<Person> lannister = new ArrayList<>();
```

Comment:
Basically this is very "wasty" (it is being called whenever this view initializes) and I can't seem to understand why you decided to go this way.